source/features/patch-diff-links.tsx

```diff
@@ -14,19 +14,23 @@ async function addPatchDiffLinks(commitMeta: HTMLElement): Promise<void> {
 	}
 
 	commitMeta!.classList.remove('no-wrap'); // #5987
-	commitMeta!.prepend(
-		<span className="sha-block" data-turbo="false">
-			<a href={`${commitUrl}.patch`} className="sha color-fg-default">patch</a>
-			{' '}
-			<a href={`${commitUrl}.diff`} className="sha color-fg-default">diff</a>
-		</span>,
-		<span className="px-2">·</span>,
+	// TODO: Improve after April 2025
+	const node = (
```

Comment:
Why?

Reply:
I mean, when the old version is deprecated we just have to go back to the old code.